src/components/molecules/TablesUserList/TablesUserList.tsx

```diff
@@ -26,9 +34,9 @@ const createTable = (i: number): Table => {
   return {
     title: `Table ${i + 1}`,
     reference: `Table ${i + 1}`,
```

Comment:
Not sure if this is in scope of this PR, if you have the capacity, please fix that. If not, I'd like you to create a bug for that.
![image](https://user-images.githubusercontent.com/63313732/125080880-583ae080-e093-11eb-87a8-d23232b7bcc9.png)

We should check if the table with the same `reference` exists already before adding the table. Because it produces weird behavior.

Reply:
@yarikoptic  Well, as far as I see every table has the unique `reference` and it could not be changed by the user (only title). Feel free to create an issue and let's check it later

Reply:
Oi, what did I do? ;-)

Reply:
That's my sorry @yarikoptic :)
 It was addressed to @sunny-viktoryia